airflow/io/__init__.py

```diff
@@ -93,11 +93,18 @@ def get_fs(
         raise ValueError(f"No filesystem registered for scheme {scheme}") from None
 
     options = storage_options or {}
+
     # MyPy does not recognize dynamic parameters inspection when we call the method, and we have to do
     # it for compatibility reasons with already released providers, that's why we need to ignore
     # mypy errors here
     parameters = inspect.signature(fs).parameters
     if len(parameters) == 1:
+        if len(options) > 0:
```

Comment:
```suggestion
        if options:
```

Reply:
Does that work with

```
    options = storage_options or {}
```
above?

Reply:
Yes since an empty dict evaluates to False

https://docs.python.org/3/library/stdtypes.html#truth-value-testing